src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--month-length--31">
+        <div class="calendar__day calendar__day--date--1 calendar__day--start-day--mon"></div>
```

Comment:
Try to do with fewer classes
Just use nth-child
```suggestion
      <div class="calendar__month calendar__month--month-length--31 calendar__day--start-day--mon">
        <div class="calendar__day"></div>
```